clientapi/routing/routing.go

```diff
@@ -685,6 +685,19 @@ func Setup(
 		}),
 	).Methods(http.MethodGet)
 
+	r0mux.Handle("/user/{userId}/openid/request_token",
```

Comment:
```suggestion
	r0mux.Handle("/user/{userID}/openid/request_token",
```
Tiny nit: We tend to use Go naming conventions for the path params.